common/src/main/java/org/keycloak/common/util/MultivaluedHashMap.java

```diff
@@ -47,7 +47,7 @@ public MultivaluedHashMap(MultivaluedHashMap<K, V> config) {
 
    public void putSingle(K key, V value)
    {
-      List<V> list = new ArrayList<V>();
+      List<V> list = new ArrayList<>();
```

Comment:
This is in the "common" module, which is used by adapters etc and require to compile on some historic java versions (JDK 7 etc). Due this, I would not use `ArrayList<>` , but rather keep this line as  `ArrayList<V>`  . As it was before

Reply:
Java 7 should support type inference for generics. I tried compiling the module with JDK 7 and it compiled fine. (There was some problem with surefire plugin but that seems unrelated.)

Reply:
Cool, Thanks! I don't know why I though it doesn't :) Approving and merging then.